### PR TITLE
Hide toolbars

### PR DESCRIPTION
**Pull request by jmercouris**
Expose FFI methods to hide toolbars.

### CODE REVIEW

File: source/window.lisp [hunk 1]
```diff
@@ -152,12 +152,19 @@ The handlers take the window as argument."))
   "Unfullscreen WINDOW, or the currently active window if unspecified."
   (ffi-window-unfullscreen window))
 
-(define-command present-current-window (&optional (window (current-window)))
+(define-command toggle-toolbar-visibility (&optional (window (current-window)))
+  "Toggle the visibility of the message and status buffer areas."
+  (if (and (zerop (ffi-window-get-status-buffer-height window))
+           (zerop (ffi-window-get-message-buffer-height window)))
```
Comment (Ambrevar) [inline source/window.lisp:158]: Nit: 
```
(= 0
   (ffi-window-get-status-buffer-height window)
   (ffi-window-get-message-buffer-height window))
```

(No need for the `and` then.)

Comment (Ambrevar) [reply source/window.lisp:158]: `toogle-toolbars` is a shorter name.

Comment (jmercouris) [reply source/window.lisp:158]: OK I change to toggle-toolbars!

**Comment by Ambrevar** [comment]
Other than that, I agree with your statement, setting the height gives us more flexibility.

What about getting and setting the size (width+height) instead with the same (renamed) FFI functions?  This would be even more generic for those would want vertical toolbars.

**Comment by jmercouris** [comment]
It would be, but it is probably a bit premature since they cannot actually change the layout of the window yet. We have some hard-coded assumptions about what goes where. Therefore, I think we should do it later. Thoughts?

**Comment by Ambrevar** [comment]
Sure, thanks!

**Pull request merged**